client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -131,109 +145,136 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
+            <Form.Group widths="equal">
+              <Form.Input
+                error={isFieldError(title)}
+                label={<h5 className="label">Pitch Title</h5>}
+                value={title}
+                onChange={(e, { value }) => setTitle(value)}
+              />
+            </Form.Group>
+            <Form.Group widths="equal">
+              <Form.Input
+                error={isFieldError(link)}
+                label={<h5 className="label">Google Doc Link</h5>}
+                value={link}
+                onChange={(e, { value }) => setLink(value)}
+              />
+            </Form.Group>
+            <Form.TextArea
+              maxLength={MAX_LENGTH}
+              value={description}
+              onChange={(e, { value }) => setDescription(toString(value))}
+              label={<h5 className="label">Description</h5>}
+              error={isFieldError(description)}
             />
-            <Form.Input
-              required
-              error={isFieldError(link)}
-              label={'Link to Google Doc'}
-              value={link}
-              onChange={(e, { value }) => setLink(value)}
-              size="small"
+            <h5 className="label-topics">
+              Associated Interests
+              <p className="unbold">(select 1-4 interests) </p>
+            </h5>
+            <InterestsSelect
+              values={interests}
+              onChange={(values) => {
+                if (values.length <= 4) {
+                  setInterests(values.map((item) => item.value));
+                } else {
+                  interestNotify();
+                }
+              }}
             />
-          </Form.Group>
-
-          <Form.TextArea
-            required
-            maxLength={MAX_LENGTH}
-            value={description}
-            onChange={(e, { value }) => setDescription(toString(value))}
-            label="Description"
-            error={isFieldError(description)}
-          />
+            {user.role === 'WRITER' ? (
+              <div>
+                {' '}
+                <h5 className="label">
+                  Do you want to be the writer of the story?{' '}
+                </h5>
+                <Form.Group inline>
+                  <Form.Radio
+                    name="writer-choice"
+                    label="Yes"
+                    onClick={() => setWriterChoice(true)}
+                    checked={writerChoice}
+                    error={isFieldError(writerChoice)}
+                  />
+                  <Form.Radio
+                    name="writer-choice"
+                    label="No, I want someone else to write the story"
+                    onClick={() => setWriterChoice(false)}
+                    checked={!isUndefined(writerChoice) && !writerChoice}
+                    error={isFieldError(writerChoice)}
+                  />
+                </Form.Group>{' '}
+              </div>
+            ) : (
+              <></>
```

Comment:
Do not render empty JSX fragments.

See this example:

````tsx

const [state, setState] = useState(false);

// return 1
return (
   
    state ? <h1>Show</h1> : <></>;
);

return (
    state && <h1>Show</h1>
    );
```

See the difference between the two returns?